readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -98,6 +98,7 @@ def get_config_params(self):
         data = {
             'current_version': self.version.verbose_name,
```

Comment:
do we still need the `current_version` in the template now?

Reply:
This is mostly for backwards compat, in case users were using it for something.